packages/libs/eda/src/lib/core/components/computations/plugins/differentialabundance.tsx

```diff
@@ -207,10 +222,54 @@ export function DifferentialAbundanceConfiguration(
     }
   }, [configuration, findEntityAndVariable]);
 
+  // If the variable is continuous, ask the backend for a list of bins
+  const continuousVariableBins = usePromise(
+    useCallback(async () => {
+      if (
+        !ContinuousVariableDataShape.is(
+          selectedComparatorVariable?.variable.dataShape
+        )
+      )
+        return;
+
+      const binRangeProps: GetBinRangesProps = {
+        studyId: studyMetadata.id,
+        ...configuration.comparator?.variable,
+        filters: filters ?? [],
+        dataClient,
+        binningMethod: 'quantile',
+      };
+      const bins = await getBinRanges(binRangeProps);
+      return bins;
+    }, [
+      dataClient,
+      configuration,
```

Comment:
I would change this to
```suggestion
      configuration.comparator?.variable,
```

As it is, a request is made when the user chooses bins in the dropdown.